src/components/Editor/HighlightLine.js

```diff
@@ -61,14 +61,15 @@ export class HighlightLine extends Component<Props> {
     const { sourceId, line } = selectedLocation;
     const editorLine = toEditorLine(sourceId, line);
 
+    if (!this.isStepping && !selectedLocation.isLocatingBreakpoint) {
```

Comment:
hmm, i'd like to find a way to avoid adding this field `selectedLocation.isLocatingBreakpoint`

sorry i dont have time to jump in and help though